pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -186,6 +186,36 @@ def test_comparison_with_unknown_scalars(self):
         tm.assert_numpy_array_equal(cat != 4,
                                     np.array([True, True, True]))
 
+    def test_comparison_of_ordered_categorical_with_nan_to_scalar(
+            self, compare_operators_no_eq_ne):
+        # https://github.com/pandas-dev/pandas/issues/26504
+        # BUG: fix ordered categorical comparison with missing values (#26504 )
+        # and following comparisons with scalars in categories with missing
+        # values should be evaluated as False
+
+        cat = Categorical([1, 2, 3, None], categories=[1, 2, 3], ordered=True)
+        scalar = 2
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", RuntimeWarning)
+            actual = getattr(cat, compare_operators_no_eq_ne)(scalar)
+            expected = getattr(np.array(cat),
```

Comment:
can u pull the expected and assert out of the context only the actual needs to be there

Reply:
Yes, good point!